lib/data.yml

```diff
@@ -6,6 +6,12 @@ dk:
     user_property: brugerejendomme
     lot_property: grunde
     lease: leje
+  sections:
+    investment: investeringsejendomme
+    user: brugerejendomme
```

Comment:
It is the section when a user want to buy location for hisself. Not for investment and it is not foreclosure